api/v1beta1/pravega.go

```diff
@@ -363,11 +363,6 @@ type Probe struct {
 }
 
 func (s *PravegaSpec) withDefaults() (changed bool) {
-	if !config.TestMode && s.ControllerReplicas < 1 {
```

Comment:
Is there a reason to stop checking for negative controller replicas?

Reply:
Yes @jkhalack there is a requirement for LTS recovery

Reply:
what if we get negative number here?

Reply:
@shshashwat we are having openapi validation, that wont allow negative numbers https://github.com/pravega/pravega-operator/blob/master/api/v1beta1/pravega.go#L157